examples/hallowelt/go_app/README.md

```diff
@@ -0,0 +1,253 @@
+# HelloWorld tutorial using Go client Fabric SDK
+This tutorial shows how to invoke Fabric Private Chaincode using the Go Fabric Client SDK and the Fabric-Private-chaincode(FPC) framework.  This assumes familiarity with the [concepts](https://hyperledger-fabric.readthedocs.io/en/release-2.0/whatis.html#), [the programming model](https://hyperledger-fabric.readthedocs.io/en/release-2.0/chaincode.html) and the [gateway] (https://hyperledger-fabric.readthedocs.io/en/release-2.0/developapps/gateway.html) in Hyperledger Fabric v2.0.
+
+## Prerequisites
+This tutorial presumes that the repository in https://github.com/hyperledger-labs/fabric-private-chaincode has been installed as per [README.md](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/README.md#requirements) in `FPC-INSTALL-DIR` which is in the `$GOPATH` folder. Additionally the [HelloWorld] example (https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/examples/README.md) has been completed.
+
+## Develop client gateway go script
+* Create a folder named `go_app`  in `FPC-INSTALL-DIR/examples/helloworld`.
+```bash
+cd $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/helloworld
+mkdir go_app
+cd go_app
+touch helloworld.go
+```
+
+* Define the go package and import the necessary libraries that will be used. `os` to access operating system variables and two libraries that enable us to access the fabric gateway and call the FPC chaincode from the gateway.
+
+```go
+package main
+
+import (
+	"os"
+
+	fpc "github.com/hyperledger-labs/fabric-private-chaincode/client_sdk/go/pkg/gateway"
+	"github.com/hyperledger-labs/fabric-private-chaincode/integration/client_sdk/go/utils"
+	"github.com/hyperledger/fabric-sdk-go/pkg/gateway"
+	"github.com/hyperledger/fabric/common/flogging"
+)
+```
+
+* We will also define several variables. `network` represents the Fabric network and `contract` our fabric private chaincode developed previously. We also are using a variable to be able to capture the errors that the different functions might return.
+
+```go
+var (
+	network  *gateway.Network
+	contract fpc.Contract
+	err      error
```

Comment:
```suggestion
```

The error is not really needed here.

Reply:
ok, will define it inside the function and see how it goes. Will do the same with Network and contract.